OptimizelySDK.Tests/OptimizelyUserContextTest.cs

```diff
@@ -63,7 +63,7 @@ public void SetUp()
         public void OptimizelyUserContextWithAttributes()
         {
             var attributes = new UserAttributes() { { "house", "GRYFFINDOR" } };
-            OptimizelyUserContext user = new OptimizelyUserContext(Optimizely, UserID, attributes, ErrorHandlerMock.Object, LoggerMock.Object);
+            var user = new OptimizelyUserContext(Optimizely, UserID, attributes, ErrorHandlerMock.Object, LoggerMock.Object);
```

Comment:
I will recommend to use interface instead of `var` across all places.

Reply:
Done